cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/turbine.json

```diff
@@ -0,0 +1,28 @@
+{
+  "project-identifier": {
+    "office-id": "LRD",
+    "location-id": "PROJECT"
+  },
+  "location": {
+    "office-id": "LRL",
+    "name": "TEST_LOCATION2",
```

Comment:
why is this "name" here when in the project identifier it is "location-id"

Reply:
good point, I'll update LocationIdentifier to use name in order to match other references.

Reply:
updated